cac-prod.yml

```
@@ -73,7 +73,7 @@ jenkins:
         title: INTESTACY Dashboard
     - buildMonitor:
         includeRegex: >-
-          HMCTS_Divorce.+\/master
+          HMCTS_Divorce/((?!div-ansible|div-case-progression-service|div-feature-toggle-client|div-frontend-boilerplate).)+\/master
```

is that better? not sure
```suggestion
          HMCTS_Divorce/(?!div-ansible|div-case-progression-service|div-feature-toggle-client|div-frontend-boilerplate).+\/master
```